jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -187,7 +187,9 @@ static final void initializeClassLoaders() {
 
 		/*[PR CMVC 193184] reflect.cache must be enabled here, otherwise performance slow down is experienced */
 		String propValue = System.internalGetProperties().getProperty("reflect.cache"); //$NON-NLS-1$
+/*[IF JAVA_SPEC_VERSION < 23]*/
```

Comment:
"java.lang.ClassLoader.lazyInitialization" on line 260 needs the same change, this one doesn't have any tests to expose the problem.

Reply:
Updated.